docs/source/package_reference/datasets.mdx

```diff
@@ -0,0 +1,8 @@
+# Datasets
+
```

Comment:
This I would put in the task as it's mostly batching logic iirc

Reply:
Done: [83043b3](https://github.com/huggingface/lighteval/pull/413/commits/83043b3f20ee97dd449e0af356f501028e5c2896)